api/src/test/java/marquez/BaseIntegrationTest.java

```diff
@@ -35,4 +44,15 @@
   protected static PostgresContainer createMarquezPostgres() {
     return PostgresContainer.create("marquez");
   }
+
+  protected CompletableFuture<HttpResponse<String>> sendLineage(String body) {
```

Comment:
_Minor_: We'll eventually want to have the open lineage call in the `marquez-java` client (using the `openlineage-java` client), but make sense to wrap the lineage call in a method for now 👍 . Another option would be to add the call to a `HttpTestUtils` class. For example: `HttpTestUtils.post(endpoint, body)`

Reply:
Noted. We can resolve in the future when we want to migrate to http2-compatible clients to minimize duplicate code.